baseapp/util_test.go

```diff
@@ -70,85 +61,6 @@ func GenesisStateWithSingleValidator(t *testing.T, codec codec.Codec, builder *r
 	return genesisState
 }
 
-func makeTestConfig() depinject.Config {
```

Comment:
I deleted that because I needed to add the runtime module in all config, so I took the liberty to clean this up as well (so by using the configurator).

Reply:
done in https://github.com/cosmos/cosmos-sdk/pull/14454